packages/vulcan-core/lib/modules/default_resolvers.js

```diff
@@ -11,18 +11,22 @@ const defaultOptions = {
   cacheMaxAge: 300,
 };
 
+// note: for some reason changing resolverOptions to "options" throws error
 export const getDefaultResolvers = (collectionName, resolverOptions = defaultOptions) => {
+  // TODO: find more reliable way to get type name from collection name
```

Comment:
Probably make any TODO comments github issues so we can link them in the code? Should make for managing these types of things easier. Can use github labels as well.

Reply:
Oh how do I do that? Just add the issue number and GitHub will link it up?

Reply:
It won't actually link it up, I'm saying when you have a TODO like this, make a github ticket about it and then put the link to the issue inside the TODO comment.

```
// TODO: find more reliable way to get type name from collection name
// https://github.com/VulcanJS/Vulcan/issues/1993
```

Reply:
good point. Maybe someone can do a search across the repo and create issues for all TODOs sometimes.